### PR TITLE
added example for Resampler

### PR DESCRIPTION
**Pull request by radioactive11**
Added example for [Resample](https://pytorch.org/audio/stable/transforms.html#torchaudio.transforms.Resample) as mentioned in [this issue](https://github.com/pytorch/audio/issues/1564)

### CODE REVIEW

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -643,6 +643,11 @@ class Resample(torch.nn.Module):
             cached as ``torch.float64``. If you use resample with lower precision, then instead of providing this
             providing this argument, please use ``Resample.to(dtype)``, so that the kernel generation is still
             carried out on ``torch.float64``.
+
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalization=True)
```
Comment (nateanl) [inline torchaudio/transforms.py:648]: nit: ``normalization`` is deprecated in ``torch.load``. Let's change it to ``normalize``

File: torchaudio/transforms.py [hunk 2]
```diff
@@ -643,6 +643,11 @@ class Resample(torch.nn.Module):
...
+        >>> resampler = transforms.Resample(sample_rate, sample_rate/10)
```
Comment (nateanl) [inline torchaudio/transforms.py:649]: Rename ``resampler`` as ``transform`` so that we have all examples consistent.

**Review by nateanl — CHANGES_REQUESTED** [review]
Thanks for the PR! We can merge it after addressing those comments.

File: torchaudio/transforms.py (hunk 2 above)
Comment (radioactive11) [reply torchaudio/transforms.py]: Sure thing!

File: torchaudio/transforms.py (hunk 1 above)
Comment (radioactive11) [reply torchaudio/transforms.py]: Thank you for the suggestion!

**Review by nateanl — APPROVED** [review]

**Comment by nateanl** [comment]
Thanks for your contribution @radioactive11! I have merged it.
